docs/netfox/nodes/rollback-synchronizer.md

```diff
@@ -57,6 +57,13 @@ func _rollback_tick(delta, tick, is_fresh):
   move_and_slide()
 ```
 
+## Single fire events
+
+The first time a tick is processed, the `is_fresh` parameter is set to `true`.
```

Comment:
I would suggest the following replacement for this line:
"In the first rollback tick processed for a node, the `is_fresh` parameter is set to `true`."